docs/modules/ROOT/pages/general-info/general-info.adoc

```diff
@@ -1,138 +1,107 @@
 [[overview]]
 = Documentation Overview
 
-This page provides some general information about the Payara Project.
+This page provides some general information about the Payara Platform Enterprise Edition.
 
 [[release-strategy]]
 == Release Strategy
 
-Payara project has a public release every quarter. Each release incorporates fixes and enhancements from the Payara team, GlassFish upstream, and the community.
+The Payara Platform Community Edition project is released each month. Each release incorporates fixes and enhancements from the Payara team, GlassFish upstream, and Enterprise only features.
 
-[[naming-strategy]]
-== Naming Strategy
+[[new-naming-strategy]]
+== New Naming Strategy (June 2020)
 
-The Payara Server naming strategy works off of the pre-existing GlassFish
-naming strategy: Append the year and quarter as the final dot version of
-the release. For example, for the Payara Server's release built on GlassFish
-4.1, released in quarter 3 of 2015, the version number would be payara-4.1.153.
+As of June 2020, The naming strategy of the Payara Platform Enterprise Edition releases is changed.
 
-In the case of updates, we will simply attach an additional point number
-to the end of the version number described above. For example, if a
-patch is released for Payara 4.1.152, the version number would be
-4.1.152.1. This will be in addition to any extra point releases that
-Oracle do for GlassFish, so it's possible for a version number to be
-something like 4.1.1.152.1!
+The general format of the version number is **Major Version Number** . **Minor Number** . **Patch Number**
 
-Releases made to date include:
+**Major Version** numbers will start from 5 (the current major version of Payara Platform) and only increment by 1 when a new major version (E.g Payara 6) is released.  Fundamental changes (e.g. Javax to Jakarta breaking change) will only be released in Major Versions. There is currently no set cadence for this version change. The change of a major version number is a product management and marketing decision.
```

Comment:
```suggestion
**Major Version** numbers will start from 5 (the current major version of Payara Platform) and only increment by 1 when a new major version (E.g Payara 6) is released.  Fundamental changes (e.g. Javax to Jakarta breaking change) will only be released in Major Versions. There is no set cadence for the major version change. 
```
I think simpler is better. No need to mention who decides on major version change.

Reply:
It was documented as such on the confluence pages.